ArcGISRuntimeSDKQt_CppSamples/Scenes/Add3DTilesLayer/README.metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "category": "Scenes",
+    "dataItems": [
+        {
+            "itemId": "00000000000000000000000000000000",
+            "path": "~/ArcGIS/Runtime/Data/3D/somedata.3tz"
+        }
+    ],
```

Comment:
this info will need to be removed if we are no longer using offline data

Reply:
I forgot to change the json file. Thanks for reminding @ldanzinger!